templates/express_oauth_microsoft/router/app.js

```diff
@@ -0,0 +1,14 @@
+import express from 'express'
+
+import { authenticateUser } from '../middleware/authenticate.js'
+
+const router = express.Router()
+
+// home page where user info is displayed
+router.get('/', authenticateUser, (req, res) => {
+    const user = req.user
+    const token = req.cookies?.token
+    res.send(`Your name is: ${user.name}<br>Your email is: ${user.email}<br>Your token is: ${token}`)
```

Comment:
Enclose in a try catch block. Catch will throw 500 and write to a log file. Refer the `express_mysql` template for doing it.